planning/obstacle_cruise_planner/src/node.cpp

```diff
@@ -75,57 +76,15 @@ bool isFrontObstacle(
   return true;
 }
 
-TrajectoryPoint calcLinearPoint(
-  const TrajectoryPoint & p_from, const TrajectoryPoint & p_to, const double length)
-{
-  TrajectoryPoint output;
-  const double dx = p_to.pose.position.x - p_from.pose.position.x;
-  const double dy = p_to.pose.position.y - p_from.pose.position.y;
-  const double norm = std::hypot(dx, dy);
-
-  output = p_to;
-  output.pose.position.x += length * dx / norm;
-  output.pose.position.y += length * dy / norm;
-
-  return output;
-}
-
-// TODO(murooka) replace with spline interpolation
 Trajectory decimateTrajectory(const Trajectory & input, const double step_length)
 {
   Trajectory output{};
 
-  if (input.points.empty()) {
+  if (input.points.size() < 2) {
```

Comment:
@purewater0901 
is this change intentional？

Reply:
Yes, this is for suppressing the warning from the resampling functions in motion utils